tests/python_functional/src/syslog_ng_config/statements/destinations/file_destination.py

```diff
@@ -28,13 +28,18 @@
 
 
 class FileDestination(DestinationDriver):
-    def __init__(self, file_name, **options):
+    def __init__(self, file_name=None, **options):
         self.driver_name = "file"
-        self.path = Path(tc_parameters.WORKING_DIR, file_name)
-        super(FileDestination, self).__init__(FileIO, [self.path], options)
+        self.options = options
+        if file_name:
+            self.positional_option = Path(tc_parameters.WORKING_DIR, file_name)
+        super(FileDestination, self).__init__(FileIO)
 
     def get_path(self):
-        return self.path
+        return self.positional_option
+
+    def set_path(self, pathname):
+        self.positional_option = Path(tc_parameters.WORKING_DIR, pathname)
```

Comment:
Thanks for the note. In the previous version if I wrote _any_ new driver (also in case of any simple driver: example_message_generator) I should know about the exists of self.positional_options variable, and I should create it even if my new driver is not using it at all. I would not like to do this. In this case drivers are using the minimal set of variables. 
Unfortunately 'aliasing' (self.path = self.positional_option) will not work here, because in case of `set_path()` we should overwrite it (`self.path = Path(...)`) and after that self.positional_option will not use the new value (which we will need in Render() class ).